src/alto-ui/Form/InputNumber/InputNumber.js

```diff
@@ -26,21 +26,34 @@ class InputNumber extends React.Component {
   }
 
   static getDerivedStateFromProps(props, state) {
-    const { value, locale, precision, currency, disableThousandSeparator, max, min } = props;
+    const {
+      value,
+      locale,
+      precision,
+      currency,
+      disableThousandSeparator,
+      percent,
+      max,
+      min,
+    } = props;
     if (
       state.prev.value !== value ||
       state.prev.precision !== precision ||
       state.prev.locale !== locale ||
       state.prev.currency !== currency ||
       state.prev.disableThousandSeparator !== disableThousandSeparator ||
+      state.prev.percent !== percent ||
       state.prev.min !== min ||
       state.prev.max !== max
     ) {
       return {
         prev: props,
         display: state.editing
           ? state.display
-          : format(value, locale, precision, currency, disableThousandSeparator, { min, max }),
+          : format(value, locale, precision, currency, disableThousandSeparator, percent, {
+              min,
+              max,
+            }),
```

Comment:
Can you put percent in the object with min and max ? The idea is to progressively put all "options" in there

Reply:
moved to options